<?xml version="1.0"?>
<launch>
    <node pkg="ultrasound_lsc" type="sonar_to_scan" name="sonar_to_scan_node" output="screen" >
	<rosparam command="load" file="$(find ultrasound_lsc)/config/sim_model.yaml" />
	<param name="frequency" value="20"/>
    </node>
</launch>
